test/workers/process_notification_event_worker_test.rb

```diff
@@ -36,6 +36,36 @@ def test_create_notifications
     end
   end
 
+  class SavingEventsTest < ActionMailer::TestCase
+    disable_transactional_fixtures!
+
+    def test_application_plan_changed_without_permissions
+      tenant = FactoryBot.create(:simple_provider)
+      services = FactoryBot.create_list(:simple_service, 2, account: tenant)
```

Comment:
I think multiple assignment can make it easier to understand the test here:

```ruby
  service_a, service_b = ...
  plan_b1, plan_b2 = ...
  member_1, member_2 = ...
  ...
  member_1.update! ...
  member_2.update! ...
```

Reply:
@guicassolato The idea is good 👍 but this was just a massive integration test to reproduce the issue and be able to fix it, it is not there in the final code 😄

Reply:
OK. Remove the commit then?

Reply:
ok 👍